codespell_lib/data/dictionary.txt

```diff
@@ -25642,6 +25656,10 @@ skeep->skip
 skelton->skeleton
 skept->skipped
 sketchs->sketches
+sketck->skate, sketch,
+sketcked->skated, sketched,
+sketckes->skates, sketches,
+sketcking->skating, sketching,
```

Comment:
Swap these round as the second term is clearly the most likely one.

Reply:
In fact I can't really see how the first term is likely in any of these cases...

Reply:
I'll swap them but i think we should keep both. I think I saw the original typo somewhere where the correct word was `sketck`.

Reply:
79a4879

Reply:
As in `sketck->skate`?

Reply:
yes. that is what I meant

Reply:
Okay fair enough.